### PR TITLE
Add version constraint and fixes for working behind transparent proxy.

### PR DESCRIPTION
**Pull request by fredrikhgrelland**

### CODE REVIEW

**Comment by zhenik** [comment]
Can you fix in this PR also pull of new vagrant box version? 
I know that release `0.0.2` has already available, but to download it, I still need to use flag `--insecure`

File: Makefile [hunk 1]
```diff
@@ -1,6 +1,8 @@
 branch = $(shell git rev-parse --abbrev-ref HEAD)
+export SSL_CERT_FILE = /etc/ssl/certs/ca-certificates.crt
+export CURL_CA_BUNDLE = /etc/ssl/certs/ca-certificates.crt
```
Comment (fredrikhgrelland) [inline Makefile:3]: This actually fies that @zhenik 
Maybe your certs are out of date. Run `aurora-ske-certificates && sudo update-ca-certificates`

Comment (zhenik) [reply Makefile:3]: Yes. It works. 
1. Would you like to add this commands to Makefile `aurora-ske-certificates && sudo update-ca-certificates` ?
2. You could also remove whitespace in exports

Comment (m88614) [reply Makefile:3]: image
works for me.

Comment (zhenik) [reply Makefile:3]: it works for me as well, after I ran `aurora-ske-certificates && sudo update-ca-certificates`

Comment (m88614) [reply Makefile:3]: I only ran `sudo update-ca-certificates`.

Comment (fredrikhgrelland) [reply Makefile:3]: I am not going to add those, as they are never needed again. (root-ca trust expired last week and a reboot should fix this.)

**Pull request merged**